projects/packages/search/src/dashboard/components/record-meter/notice-box.jsx

```diff
@@ -30,32 +31,22 @@ const getNotices = ( tierMaximumRecords = null ) => {
 		},
 		3: {
 			id: 3,
-			message: __(
-				"We weren't able to locate any content for Search to index. Perhaps you don't yet have any posts or pages?",
+			header: __(
+				"We weren't able to locate any content for Search to index",
 				'jetpack-search-pkg'
 			),
+			message: __( "Perhaps you don't yet have any posts or pages?", 'jetpack-search-pkg' ),
 		},
 		4: {
-			id: 4,
-			message: sprintf(
-				// translators: %s: site's current plan record limit
-				__(
-					'You recently surpassed %d records and will be automatically upgraded to the next billing tier <p> learn more <p>',
-					'jetpack-search-pkg'
-				),
-				tierMaximumRecords
-			),
-			link: {
-				text: __( 'learn more', 'jetpack-search-pkg' ),
-				url: 'https://jetpack.com/support/search/product-pricing/',
-			},
-		},
-		5: {
 			id: 5,
+			header: __(
+				"You're close to the max amount of records for this billing tier",
+				'jetpack-search-pkg'
+			),
 			message: sprintf(
 				// translators: %s: site's current plan record limit
 				__(
-					"You're close to the max amount of records for this billing tier. Once you hit %s indexed records, you'll automatically be billed for <br> the next tier <p> learn more <p>",
+					"Once you hit %s indexed records, you'll automatically be billed for the next tier",
```

Comment:
In the designs, the message body always seems to end with a full stop.
```suggestion
					"Once you hit %s indexed records, you'll automatically be billed for the next tier.",
```